daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Profile.scala

```diff
@@ -92,9 +91,7 @@ object Profile {
     ) {
       def write(path: Path): Unit = {
         import JsonProtocol.fileFormat
-        val writer = new BufferedWriter(new FileWriter(path.toFile()))
-        writer.write(this.toJson.compactPrint)
-        writer.close()
+        val _ = Files.write(path, this.toJson.compactPrint.getBytes)
```

Comment:
`getBytes` clones the string. I'd suggest using `Seq(this.toJson.compactPrint).asJava` instead.

Reply:
I wasn't aware of that. Thanks for pointing it out. I changed it accordingly.